Source/CNTKv2LibraryDll/proto/onnx/CNTKToONNX.cpp

```diff
@@ -5277,6 +5281,29 @@ bool CNTKToONNXHelper::ProcessLoopsAndCheckCNTKNodeContinueCreate(const Function
                             graph->AddInitializedTensor(scanLoopState.m_initialStateTensor);
                     }
                     // else initializer is input.
+
+                    // FutureValue's custom attrubute will be serialized into the Scan node's description
+                    auto fvOp = scanLoopState.m_stateOutput.Owner();
+                    if (fvOp && fvOp->OpName() == L"FutureValue")
+                    {
+                        const Dictionary& dict = fvOp->GetCustomAttributes();
+                        if (dict.Size() > 0)
+                        {
+                            if (futureValueCustomAttrStr == "")
+                            {
+                                futureValueCustomAttrStr = "{custom_attributes:" + SerializeDictionaryToString(dict) + "}";
```

Comment:
json dict keys as string needs to be quoted. Please use "{\"custom_attributes\":"